src/www/templates/group/groupDetails.html

```diff
@@ -1,10 +1,28 @@
 <!--TODO Abstract from container-->
 <div class="container">
-    <h1>{{group.displayName}}</h1>
-    <h4>Group description</h4>
-    <p>{{group.description}}</p>
-    <h4>Contact details</h4>
-    <p>{{group.email}}</p>
-    <h4>Members</h4>
-    <p ng-repeat="member in group.members">{{member.displayName}} ({{member.user_group.func}})</p>
+    <h1>
+        <a class="btn btn-link" href="javascript:history.back()">
+            <i class="fa fa-chevron-left" aria-hidden="true"></i>
+        </a>
+        {{group.displayName}}</h1>
+    <div class="row HA-col-container">
+        <div class="col-sm-6 col-md-6">
+            <h2>{{group.fullName}}</h2>
+            <p>{{group.description}}</p>
+            <i>Since {{group.createdAt | date: 'dd MMM, yyyy'}}</i>
+            <h4>Contact details</h4>
+            <p><a href="mailto:{{group.email}}"><i class="fa fa-envelope" aria-hidden="true"></i> {{group.email}}</a></p>
+            <h4>Current members</h4>
+            <p ng-repeat="member in group.members">{{member.displayName}} ({{member.user_group.func}})</p>
+            <div class="no-committees" ng-if="!group.members.length">
+                Members are gone 🦀🦀🦀.
```

Comment:
Crabs?

Reply:
Especially as this is likely to occur when a new committee is created, I would make it a bit more formal.